README.md

```diff
@@ -27,7 +27,7 @@ Features
 Supported FreeIPA Versions
 --------------------------
 
-FreeIPA versions 4.6 and up are supported by all roles. 
+FreeIPA versions 4.6 and up are supported by all roles.
```

Comment:
This white space change is not related.

Reply:
Fixed. This is something with my text editor.